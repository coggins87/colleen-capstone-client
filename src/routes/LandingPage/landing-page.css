.Landing_page {
  display: inherit;
  flex-direction: column;
  align-content: space-around;
  text-align: center;
  padding-top: 11%;
  background-color: rgb(20, 5, 9);

}

.heading_page {
  display: inherit;
  flex-direction: inherit;
  align-content: flex-end;
  margin-top: -5%;
  color: white;
  background-color: rgb(20, 5, 9);
  padding-top: 8%;
  padding: 1%;
}
#landing-img{
  height:auto;
  width:100vw;
}
.landing_title {
  font-family: "Monda", sans-serif;
  margin-bottom: 12px;
  text-shadow: 2px 2px #ff0000;

}
.subheader {
 padding: 0px;
  margin: 0;
  color: whitesmoke;
  font-weight: 500
}
.app_desc {
  list-style-type: none;
  padding: 1%;
  margin: 0 auto;
}
ul {
  padding-left: 0px;
}
li {
  padding: 10px;
}
a {
  color:white;
}
a:hover{
  color:purple;
}
.definition {
  max-width: 500px;
  text-align: left;
  margin: 0 auto;
  padding: 2%;
}

p {
  margin: 0px;
}

@media all and (min-width: 1001px) {
  .landing_title {
    font-size: 3em;
  }
  .Landing_page {
    padding-top: 10%;
  }
  .heading_page {
    height: 100%;
    margin-top: -6%;
  }
  .app_desc,
  .definition {
    width: 80%;
    justify-self: center;
  }
}
@media all and (max-width: 1000px) {
  .heading_page {
    background-size: cover;
    min-height: 200px;
  }
  .app_desc,
  .definition {
    width: 80%;
    justify-self: center;
  }
}

@media all and (max-width: 800px) {
 
  .heading_page {
    background-size: cover;
    min-height: 200px;
    padding-top: 5%;
  }
}

@media all and (max-width: 509px) {
  .Landing_page {
    margin-top: 22%;
  }
  .heading_page {
    background-size: cover;
    min-height: 200px;
  }
}
@media all and (max-width: 350px) {
  .Landing_page {
    margin-top: 35%;
  }
  .heading_page {
    background-size: cover;
    min-height: 200px;
  }
}

.error_button {
  border-style: none;
  font-weight: bold;
}