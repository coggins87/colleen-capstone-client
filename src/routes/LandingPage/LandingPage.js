import React from 'react'
import ('./landing-page.css')
export default function LandingPage () {
  return (
    <div className='Landing_page'>
      <div className='heading_page'>
      <h2 className="landing_title">Create randomly generated AMRAP workouts</h2>
      <li className='subheader'><h4>From a database of movements curated by a licensed physical therapist</h4></li>

      </div>
      <div className ='info_'>
      <ul className ="app_desc">

      <li>Click 'Generate A Workout' to search for a new AMRAP workout</li>
      <li>Create an account or log in to save workouts for later!</li>
      </ul>
      <div className="definition">
      <p className="def_word">AMRAP[amrap]:</p>
<p>     noun
      <br></br>1. An abbreviation that means 'as many reps as possible'<br></br>2. To complete a circuit of exercises as many times as possible within a specific period of time</p>
    </div>
    </div>
    </div>
  )
}