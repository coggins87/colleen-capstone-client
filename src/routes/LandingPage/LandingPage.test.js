import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingPage from './LandingPage'

describe(`LandingPage component`, () => {


  it('renders the LandingPage component by default', () => {
    const wrapper = shallow(<LandingPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})