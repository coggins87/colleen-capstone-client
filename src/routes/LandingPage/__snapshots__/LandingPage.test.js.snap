// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the LandingPage component by default 1`] = `
<div
  className="Landing_page"
>
  <div
    className="heading_page"
  >
    <h2
      className="landing_title animated fadeIn"
    >
      Create randomly generated AMRAP workouts
    </h2>
    <h4
      className="subheader animated fadeIn delay-3s"
    >
      From a database of movements curated by a licensed physical therapist
    </h4>
    <ul
      className="app_desc"
    >
      <div
        role="alert"
      />
      <li
        className="animated fadeIn delay-4s"
      >
        Click 'Generate A Workout' to search for a new AMRAP workout
      </li>
      <li
        className="animated fadeIn delay-5s"
      >
        <Link
          to="/register"
        >
          Create an account
        </Link>
         or 
        <Link
          to="/login"
        >
          log in
        </Link>
         to save workouts for later!
      </li>
    </ul>
  </div>
  <img
    alt="women in workout clothes in a starting position"
    id="landing-img"
    src="startingline.jpg"
  />
</div>
`;
